### ISSUE TITLE
Only use training data for kmeans in RunExample

### ISSUE THREAD

**Issue by mguldner** [mguldner/Data_Algorithms#2]
For now, I combine the two datasets because some value for fare in test set are higher than the highest value in training set.
Solutions : 
* if higher value, add a new Frame with top bound is this new value ? And if already an other value in test set that has this behavior, do not add a new Frame but just increase this one? But problem because it's also impossible to use the trained model to cluster these values...

* if higher value, put in Frame -1 <= x <= -1. But loss of interest...
